APIURLS.md

```diff
@@ -0,0 +1,571 @@
+		            "email": "user1@example.com"
+		        },
+		        {
+		            "_id": "userId2",
+		            "name": "User Name 2",
+		            "email": "user2@example.com"
+		        }
+		    ]
+		}
+        User status:
+            GET localhost:8080/API/v1/Users/Userid?status
+     	    Response:
+		{
+	    		"status": "online"
+		}
+        Update User Last active:
+            GET localhost:8080/API/v1/Users/UserId?lastActive=datetime
+     	    Response:
+		{
+	    		"last_active": "datetime"
+		}
+
+            PATCH localhost:8080/API/v1/Users/UserId?LastActive=datetime
+	    Request:
+     		PATCH /API/v1/Users/{UserId} HTTP/1.1
+		Host: localhost:8080
+		Content-Type: application/json
+		Authorization: Bearer your_access_token
+		
+		{
+		    "last_active": "2024-04-12T15:00:00Z"
+		}
+     	    Response:
+	  	{
+		    "status": "success",
+		    "msg": "Last active time updated successfully."
+		}
+        User Detailed Name Card:
+            GET localhost:8080/API/v1/Users?UserId=1&fullname=zhiyuanguo&emailaddress=example@exm.com&publicname=Lambton42&initials=zy&status=online&lastactive=datetime
+	    Response:
+		{
+		    "_id": "1",
+		    "full_name": "Zhiyuan Guo",
+		    "public_name": "Lambton42",
+		    "initials": "zy",
+		    "email": "example@exm.com",
+		    "status": "online",
+		    "last_active": "2024-04-12T15:00:00Z"
+		}
+    Task Board Management:
+        Tasks Database:
+            localhost:8080/API/v1/task/task_id/1
+            [Post, Put, Get, Delete]
+            localhost:8080/API/v1/tasks
+        Task Search:
+	    GET localhost:8080/API/v1/tasks?task_id=1
+     	    Response:
+		{
+		  "_id": "uniqueIdHere",
+		  "name": "Task Title",
+		  "description": "Task Description",
+		  "total_order_id": "projectIdHere",
+		  "assignee": "userIdHere",
+		  "reporter": "userIdHere",
+		  "priority": "high",
+		  "start_date": "2024-04-05T09:00:00",
+		  "due_date": "2024-04-10T09:00:00",
+		  "status_id": "status_id",
+		  "activity": {
+		    "comments": [
+		      {
+		        "_id": "uniqueIdHere",
+		        "timestamp": "2024-04-05T09:15:00",
+		        "user_id": "userId",
+		        "Initials": "JD",
+		        "comment": "This is a sample comment."
+		      }
+		    ],
+		    "history": [
+		      {
+		        "_id": "uniqueIdHere",
+		        "timestamp": "2024-04-05T09:30:00",
+		        "user": "userIdHere",
+		        "Initials": "JD",
+		        "event": "Task created"
+		      }
+		    ],
+		    "actions": [
+		      {
+		        "link issue": [
+		          {
+		            "_id": "uniqueIdHere",
+		            "timestamp": "2024-04-07T11:30:00Z",
+		            "link_id": "projectIdHere"
+		          }
+		        ],
+		        "attached file": [
+		          {
+		            "_id": "uniqueIdHere",
+		            "timestamp": "2024-04-08T14:00:00",
+		            "file_name": "example.txt"
+		          }
+		        ]
+		      }
+		    ]
+		  }
+		}
+        Task Filter:

```

Comment:
Missing the `SearchAPI` based on the search box input

Reply:
Has this one addressed? I don't see the API